lib/elixir_analyzer/test_suite/name_badge.ex

```diff
@@ -0,0 +1,17 @@
+defmodule ElixirAnalyzer.TestSuite.NameBadge do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise NameBadge
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+  alias ElixirAnalyzer.Constants
+
+  feature "Use an if macro" do
+    type :essential
+    comment Constants.name_badge_use_if()
+
+    form do
+      if(_ignore, do: _ignore, else: _ignore)
```

Comment:
I wonder if we should allow either `if` with `do` or `if` with `do` and `else`? It's probably pretty hard to solve it with just `if`s without `else`s, but I worry about user frustration because the comment (that I just merged...) doesn't say that you need an `else` for this check to pass 😁

Reply:
I actually started with a version that allowed `if` without `else` but ended up getting rid of it at the last minute (I didn't have good tests for it). I'll put it back in then :)